frontend/src/static/sideNav.json

```diff
@@ -13,6 +13,12 @@
                 "icon": "virtual-column--key",
                 "link": "/grant",
                 "disabled": true
+            },
+            {
+                "name": "Add application admin",
+                "icon": "enterprise",
+                "link": "#",
```

Comment:
and this link? I can get the grant access page

Reply:
sure, I'll do

Reply:
sorry please ignore my comment here, you're right, it's not implemented yet